components/service/glean/docs/pings/metrics.md

```diff
@@ -2,18 +2,30 @@
 
 ## Description
 The `metrics` ping is intended for all of the metrics that are explicitly set by the application or
-are included in the application's `metrics.yaml` file.
+are included in the application's `metrics.yaml` file. The reported data is tied to the ping's
+*measurement window*, which is the time between the collection of two `metrics` ping. Ideally, this
+window is expected to be about 24 hours, given that the collection is scheduled daily at 4AM. Data
+in the [`ping_info`](pings.md#the-ping_info-section) section of the ping can be used to infer the
+length of this window.
 
 ## Scheduling
-The `metrics` ping is automatically sent when the application enters the [background](pings.md#defining-background-state)
+The desired behaviour is to collect the ping at the first available opportunity after 4AM local
+time on a new calendar day. This breaks down in two scenarios:
 
-It is also required that the throttling time interval has elapsed, which is currently 24 hours.  
-This limits `metrics` pings to be sent not more than once per day. The schedule is only checked 
-when the application enters the background state so pings may be sent after an interval longer 
-than 24 hours but never shorter than 24 hours.
+1. the application was just started (after a crash or a long inactivity period);
+2. the application was open and the 4AM due time was hit.
+
+In the first case, if the `metrics` ping was already collected on the current calendar day, a new
+collection will be scheduled for the next calendar day, at 4AM. If no collection already happened,
+and the due time for the current calendar day has passed, a `metrics` ping is immediately generated
+and scheduled for sending.
+
+In the second case, similarly to the previous case, if the `metrics` ping was already collected once
```

Comment:
"was already collected"? This seems ambiguous. Does that mean on the current calendar day?